src/atom_finder/classifier/operator-precedence.clj

```diff
@@ -51,9 +51,16 @@
 (defn specific-confusing-operator-combination?
   "if the combination of groups of operators exists in this set, then the combination is confusing"
   [combination]
-  (some #(= % combination) (map sort #{[:de_incr :pointer] [:multiply :add] [:arith_unary :add] [:arith_unary :multiply] [:and :add] [:and :multiply] [:and :arith_unary] [:or :add] [:or :multiply] [:or :arith_unary] [:or :and] [:not :add] [:not :multiply] [:not :arith_unary] [:not :and] [:not :or]
-;;[:compare :and] [:compare :or] // Underclassify
-[:compare :not] [:compare :compare] [:pointer :add] [:cond :arith_unary] [:cond :and] [:cond :or] [:cond :not] [:cond :compare] [:cond :cond] [:non-asso :add] [:non-asso :multiply] [:non-asso :arith_unary] [:non-asso :and] [:non-asso :or] [:non-asso :not] [:non-asso :non-asso] [:field_ref :pointer]})))
+  (some #(= % combination) (map sort [[:de_incr :pointer] [:multiply :add] [:arith_unary :add] [:arith_unary :multiply]
```

Comment:
Are you sure it's not _wrong_ to not sort `combination` I feel like it'll actually get the wrong answer 50% of the time?

Also, this function has to do 60 comparisons or something. But if you put all the operator combinations into a set and store that using `def`, then use a set lookup, it's only a single comparison. 60x speedup.

    (def confusing-combinations (into #{} (map sort [[:de_incr :pointer] [:multiply :add] ...])))
    (defn specific-confusing-operator-combination? [combination] (confusing-combinations (sort combination)))